notebooks/239-image-bind/README.md

```diff
@@ -10,13 +10,26 @@ In this tutorial, we consider how to convert and run ImageBind model using OpenV
 
 ## Notebook Contents
 
-The tutorial consists of following steps:
+This folder contains two notebooks that show how to optimize and quantize model with OpenVINO:
+
+1. [Optimize an ImageBind Model using OpenVINO](239-image-bind.ipynb)
+2. [Quantize an ImageBind Model using NNCF](239-image-bind-quantize-nncf.ipynb)
+
+The optimization tutorial consists of following steps:
 
 1. Download the pre-trained model.
 2. Prepare input data examples.
 3. Convert the model to OpenVINO Intermediate Representation format (IR).
 4. Run model inference and analyze results.
 
+NNCF performs quantization within the OpenVINO IR. It is required to run the optimization notebook before running the quantization tutorial.
+
+This quantization tutorial consists of the following steps:
+
+1. Quantize the converted OpenVINO model with the Post-training Quantization with NNCF Tool API in OpenVINO.
+2. Compare results of the converted OpenVINO model and the quantized model.
+3. Compare performance of the converted OpenVINO model and the quantized model.
```

Comment:
can we also add model size?

Reply:
model size comparision added in readme but missed in table of content of notebook, could you please align?